### ISSUE TITLE
Long prefixes on mobile 

### ISSUE THREAD

**Issue by vadi2** [demonnic/build-with-muddler#6]
The action works pretty great and fast, but the logs are a bit hard to review on mobile because the muddler and info prefixes take up space:

![Screenshot_2023-12-12-10-26-58-37_3aea4af51f236e4932235fdada7d1643](https://github.com/demonnic/build-with-muddler/assets/110988/3308478b-ee65-459d-8ee4-03cb1f63cd2d)

**Comment by demonnic** [comment]
hmmm. This is just the output of muddler being passed along. I could look at maybe tweaking Muddler's output, either add a terse mode or make the existing a verbose mode and output a much quieter version by default.

**Comment by vadi2** [comment]
Would it be possible to remove `MUDDLER: [INFO]: ` prefix from every line, or not much we can do?

**Comment by demonnic** [comment]
yeah, just have to decide if I just wanna remove that altogether, or add quiet or verbose or what. Will knock it out whichever I decide on in the next couple days.
